### PR TITLE
NL eval code rename and clean up to clear confusion

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/apps/nl_eval/model_match_box.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ function findKNearestEmbeddings(
   return result.slice(0, k);
 }
 
-export function SearchResult(props: SearchResultProps): JSX.Element {
+export function ModelMatchBox(props: ModelMatchBoxProps): JSX.Element {
```
Comment (chejennifer) [inline static/js/apps/nl_eval/model_match_box.tsx:65]: edit: maybe ModelScoreSection? ModelMatch makes me think we're matching things to the models

File: static/js/apps/nl_eval/one_query_section.tsx [hunk 2]
```diff
@@ -16,29 +16,29 @@
...
-export interface SentenceSectionProps {
+export interface OneQuerySectionProps {
   sentence: string;
   modelNames: string[];
   goldenStatVars: string[];
   customDescription: Record<string, EmbeddingObject[]>;
 }
 
-export function SentenceSection(props: SentenceSectionProps): JSX.Element {
+export function OneQuerySection(props: OneQuerySectionProps): JSX.Element {
```
Comment (chejennifer) [inline static/js/apps/nl_eval/one_query_section.tsx:29]: nit: curious why OneQuerySection, will there be another MultiQuerySection? If not, could this just be QuerySection?

**Review by chejennifer — APPROVED** [review]
nice! Some nits/suggestions on naming

File: static/js/apps/nl_eval/model_match_box.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/apps/nl_eval/model_match_box.tsx]: changed

File: static/js/apps/nl_eval/one_query_section.tsx (hunk 2 above)
Comment (shifucun) [reply static/js/apps/nl_eval/one_query_section.tsx]: Updated

**Pull request merged**